crates/gpui/src/platform/cosmic_text/text_system.rs

```diff
@@ -210,9 +210,16 @@ impl CosmicTextSystemState {
     #[profiling::function]
     fn load_family(
         &mut self,
-        name: &SharedString,
+        name: &str,
         _features: FontFeatures,
     ) -> Result<SmallVec<[FontId; 4]>> {
+        // TODO: Determine the proper system UI font.
```

Comment:
A lot of distros seem to converge on Noto Sans these days. Alternatively (if only fontconfig aliases got resolved properly) the abstract "sans-serif" family could be used to pick up whatever the user has configured in their fonts.conf. (which unfortunately is not in sync with what is configured in the GNOME/KDE appearance settings...)

Of course sticking with Zed Sans as a conscious style choice is always option.

Reply:
This PR simply renames the linux text system to the cosmic text system and uses it on both windows and linux.

This TODO is possibly not relevant unless you're worried about something negatively impacting the Windows usage?

Reply:
Ah, true. I must've been looking at the old name. :+1: